scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -863,14 +866,15 @@ public void onResponseReceived(Request request, Response response) {
 
             @Override
             public void onError(Request request, Throwable exception) {
-                GWT.log("Error occured when fetching buckets from Catalog");
+                LogModel.getInstance().logImportantMessage("Error occured when fetching buckets from Catalog");
             }
         });
 
         try {
             req.send();
         } catch (RequestException e) {
-            GWT.log("Error occured when fetching buckets from Catalog");
+            LogModel.getInstance().logImportantMessage("Error occured when fetching buckets from Catalog");
+            // For Dev debug mode only
             e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)